test/drag/drag.js

```diff
@@ -1,5 +1,5 @@
-//load a lot of stuff ...
-steal(function(){
+steal("../../browser/jquery", "../../node_modules/jquerypp/index", function($){
```

Comment:
It'd probably be better to "require" these files so we're more consistent with other files.